src/robusta/core/sinks/timing.py

```diff
@@ -0,0 +1,78 @@
+from datetime import datetime
+from typing import List, Tuple
+
+import pytz
+
+from robusta.core.model.env_vars import DEFAULT_TIMEZONE
+
+
+_DAY_STR_TO_NUM = {
+    "MON": 0,
+    "TUE": 1,
+    "WED": 2,
+    "THR": 3,
+    "FRI": 4,
+    "SAT": 5,
+    "SUN": 6,
+}
+
+DAY_NAMES = list(_DAY_STR_TO_NUM.keys())
+
+
+class TimeSliceBase:
+    def is_active_now(self) -> bool:
+        raise NotImplementedError()
+
+
+class TimeSlice(TimeSliceBase):
+    def __init__(self, days: List[str] = None, time_intervals: List[Tuple[str, str]] = [], timezone=DEFAULT_TIMEZONE):
```

Comment:
I think days shouldn't have default value
If someone calls it without days, line 29 throws an error

Reply:
Indeed!